src/utils/components/CoordinateCopySlot.vue

```diff
@@ -90,20 +110,21 @@ async function copyValue() {
     </div>
     <div class="location-popup-data">
         <div>
-            <div :data-cy="`location-popup-${identifier}`">
-                {{ value }}
+            <div :data-cy="`${identifier}`">
+                {{ display(value) }}
             </div>
-            <div v-if="extraValue" :data-cy="`location-popup-extra-value-${identifier}`">
+            <div v-if="extraValue" :data-cy="`${identifier}-extra-value`">
                 {{ extraValue }}
             </div>
         </div>
         <button
             ref="copyButton"
-            class="location-popup-copy-button btn btn-light text-black-50 d-none d-md-block"
+            :data-cy="`${identifier}-button`"
+            class="location-popup-copy-button btn btn-light text-black-50"
```

Comment:
a margin or padding between the coordinate and the button should be added for the FeatureDetail.vue and FeatureStyleEdit.vue case